app/__init__.py

```diff
@@ -1,28 +1,21 @@
-import os
-
 from flask import Flask
-from flask_migrate import Migrate
 from flask_jwt_extended import JWTManager
+from flask_migrate import Migrate
```

Comment:
Looks like the code associated with this import is redundant here. Also, just a reminder scripts should be handled in your script manager i.e. `manage.py` file